packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -99,6 +103,18 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
 
   public componentDidMount(): void {
     this._fitParentContainer();
+    if (this.props.showYAxisLables) {
+      const maxYAxisLabelLength = calculateLongestYAxisLabel(this.props.points, this.props.theme!);
+      if (this.state.startFromX !== maxYAxisLabelLength) {
```

Comment:
what if startFromX is greater than maxYAxisLabelLength

Reply:
It can never be grater than maxYAxisLabelLength. Please note we are calculating max of length of all y lables using calculateLongestYAxisLabel().